### DISCUSSION TITLE
Speech Recognition based on Edge Impulse | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1331] [Q&A]
# Speech Recognition based on Edge Impulse | Seeed Studio Wiki

Speech Recognition based on Edge Impulse

https://wiki.seeedstudio.com/XIAO-BLE-Sense-PDM-Usage/

**Comment by LumingLin** [comment]
I got this error while uploading the code:

Invalid library found in /Users/user/Library/Arduino15/packages/Seeeduino/hardware/mbed/2.9.2/libraries/Spi_Flash: no headers files (.h) found in /Users/user/Library/Arduino15/packages/Seeeduino/hardware/mbed/2.9.2/libraries/Spi_Flash
